roles/patch-vulns/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- name: Remove known log4j files / folders
+  become: yes
+  become_user: root
+  file:
+    path: "{{ item }}"
+    state: absent
+  with_items:
+    - "{{ grid_home }}/tfa/{{ inventory_hostname_short }}/tfa_home/jlib/log4j-core-2.9.1.jar"
+    - "{{ grid_home }}/tfa/{{ inventory_hostname_short }}/tfa_home/jlib/tfa.war"
+    - "{{ grid_home }}/suptools/tfa/release/tfa_home/jlib/log4j-core-2.9.1.jar"
+    - "{{ grid_home }}/suptools/tfa/release/tfa_home/jlib/tfa.war"
+    - "{{ grid_home }}/.patch_storage"
```

Comment:
If all of .patch_storage is removed, users probably won't ever be able to roll back the patch.  Is it possible to just remove the specific files that are known to be an issue?

Reply:
Done

Adding:
```

$GRID_HOME's patch backup files:
.patch_storage/34419443_Oct_14_2022_05_25_14/files/suptools/tfa.zip
-----------
$ORACLE_HOME's patch backup files:
.patch_storage/34419443_Oct_14_2022_05_25_14/files/md/property_graph.zip
.patch_storage/34419443_Oct_14_2022_05_25_14/files/suptools/tfa.zip

```


Note:
Excerpting from (internal): https://b.corp.google.com/issues/265034396#comment21
> https://paste.googleplex.com/5242743847649280 ==> has the full `superverify` output after the `19.18` run, but the gist is that there is no `.patch_storage/*` file that's vulnerable:
>
```
[root@linuxserver44 ~]#  curl https://www.gstatic.com/log4j_jarscanner/2hamster-scanner-linux -o jar-scanner; chmod +x jar-scanner; ./jar-scanner
...
Found vulnerable JARs:
/u01/oracle_install/V982063-01.zip
/u01/oracle_install/V982068-01.zip
```

Reply:
Looks good, thanks